packages/components/README.md

```diff
@@ -6,56 +6,67 @@ Reusable UI components for Deriv.
 
 **In this document**
 
--   [Other documents](#other-documents)
 -   [Commands](#commands)
 -   [Storybook](#storybook)
 -   [Usage](#usage)
 
-## Other documents
-
--   [General](docs/README.md) - Contains general philosophy and overview of this package
--   [Modules docs](docs/Modules/README.md) - Contains implementation guides (i.e., scaffolding, code usage)
-
 ## Commands
 
 Setup:
+The `npm run bootstrap` command should be run in the root of the project, because it does not exist in the components package.js
 
 ```sh
-lerna bootstrap
+npm run bootstrap
 ```
 
 Serve:
+The `npm run serve` command is used to serve the components package
```

Comment:
Shouldn't be like this? `npm run serve components`

Reply:
Hello @niloo-fs, 
Yes indeed, it should be

Reply:
@niloo-fs, I actually had divided attention yesterday due to BeSquare, but this is not right. Remember all the commands here are being run against the package.json in the components folder, so `npm run serve` is the valid one and not `npm run serve components` as this is run against the package.json in the root folder. Therefore, I'll be undoing this change.